packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -96,9 +96,21 @@ export const getCallCompositePage = (
   // Must check for ongoing call *before* looking at any previous calls.
   // If the composite completes one call and joins another, the previous calls
   // will be populated, but not relevant for determining the page.
+  call?.remoteParticipants;
   if (call) {
+    // For PSTN/ACS Calls, the composite should display the call screen immediately.
+    // Currently there is no way to determine the type of a call (group, 1:1, PSTN etc)
+    // For PSTN and ACS calls, when you call a single user, the call state differs from when you call multiple users.
+    // For example, When you call a single PSTN user, the call state will be `Connecting | Ringing` until user picks up.
```

Comment:
When is it `connecting` and and when is it `ringing`? Does it go:
`Idle -> Connecting -> Ringing -> (someone picks up) -> Connected`
or
`Idle  -> Ringing -> (someone picks up) -> Connecting -> Connected`

Reply:
its the first one, Connecting state is part of when we are joining the call, you can follow it in this flow chart found here: https://docs.microsoft.com/en-us/azure/communication-services/how-tos/cte-calling-sdk/manage-calls
![image](https://user-images.githubusercontent.com/94866715/183219277-727c46af-68d6-46f3-8435-e60b68438f08.png)

Reply:
Please add a link to this docs page in the docstring for this function, or as a comment. I'm sure I'll need to refer to it the next time I'm trying to modify this function.